ts/receiver/closedGroups.ts

```diff
@@ -907,7 +906,11 @@ export async function createClosedGroup(groupName: string, members: Array<string
   // Subscribe to this group id
   SwarmPolling.getInstance().addGroupId(new PubKey(groupPublicKey));
 
-  await Promise.all(promises);
+  let results = await Promise.all(promises);
```

Comment:
change the `getMessageQueue().sendToPubKey` to a  `getMessageQueue().sendToPubKeyNonDurably` instead here